module/VuFind/src/VuFind/ILS/Driver/SierraRest.php

```diff
@@ -247,6 +260,10 @@ public function init()
 
         if (isset($this->config['Catalog']['api_version'])) {
             $this->apiVersion = $this->config['Catalog']['api_version'];
+            // Default to API v5 unless a lower compatibility level is needed.
+            if ($this->apiVersion < 5) {
+                $this->apiBase = 'v' . substr($this->apiVersion, 0, 1);
```

Comment:
Would `floor` be better than `substr` here to avoid problems if we eventually have two-digit version numbers?

Reply:
Changed. It's cleaner now, but since this is only done if api version is < 5, there wasn't much of a risk.

Reply:
Ahh, yes, of course! Thanks for adjusting, in any case. :-)